### PR TITLE
Fix API server cert when external address is numeric

### PR DESCRIPTION
**Pull request by csrwng**
Fixes generated certs when the external address is numeric. With this, hypershift install works on GCP.

### CODE REVIEW

File: pkg/pki/openshift.go [hunk 1]
```diff
@@ -32,22 +32,32 @@ func GeneratePKI(params *api.ClusterParams, outputDir string) error {
...
+		"kubernetes",
+		"kubernetes.default.svc",
+		"kubernetes.default.svc.cluster.local",
+		"kube-apiserver",
+		fmt.Sprintf("kube-apiserver.%s.svc", params.Namespace),
+		fmt.Sprintf("kube-apiserver.%s.svc.cluster.local", params.Namespace),
+	}
+	apiServerIPs := []string{
+		kubeIP.String(),
+		params.ExternalAPIIPAddress,
```
Comment (joelddiaz) [inline pkg/pki/openshift.go:45]: don't you want this removed, and only added when line 47 falls through that condition?

Comment (csrwng) [reply pkg/pki/openshift.go:45]: It's different... this is the ExternalAPI**IP**Address, which I always want in the IPs list

Comment (joelddiaz) [reply pkg/pki/openshift.go:45]: ah yes, i see it now

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Pull request merged**